source/MapDetailPanel.cpp

```diff
@@ -288,6 +321,7 @@ bool MapDetailPanel::Click(int x, int y, int clicks)
 	// If the system just changed, the selected planet is no longer valid.
 	if(selectedPlanet && !selectedPlanet->IsInSystem(selectedSystem))
 		selectedPlanet = nullptr;
+
```

Comment:
Empty line.

Reply:
Yeah I added it because it adds clarity I think but whatever I don't mind it